### PR TITLE
Fix dependency vulnerabilities

### PR DESCRIPTION
**Pull request by russelarms**
Things to consider before you submit the PR:

* [x] Are the files formatted correctly?
* [x] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

Fix dependency vulnerabilities.

## Related PRs or issues

https://github.com/microsoft/appcenter-sdk-react-native/issues/743
https://snyk.io/test/npm/debug/2.2.0

[AB#69172](https://msmobilecenter.visualstudio.com/454a20a9-dfe6-4c1f-8ae8-417f76adb472/_workitems/edit/69172)

### CODE REVIEW

**Comment by guperrot** [comment]
Did you test linking in a blank app on RN 0.59?

**Comment by dhei** [comment]
It's unlikely to break anything because it's only a logging tool, but it doesn't hurt to test on a blank app.

File: appcenter-link-scripts/package.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
   },
   "homepage": "https://github.com/microsoft/appcenter-sdk-react-native#readme",
   "dependencies": {
-    "debug": "2.2.0",
+    "debug": "2.6.9",
```
Comment (dhei) [inline appcenter-link-scripts/package.json]: In theory, the vulnerable package could be used in second level or third level dependencies in package.json. Therefore, to ensure we don't consume any of them, we should search for the unsecure version of the package in TestApp/package-lock.json and DemoApp/package-lock.json. 
Note that we don't checkin package-lock.json files but they are used by GitHub to detect vulnerabilities displayed in security alerts at https://github.com/microsoft/appcenter-sdk-react-native/network/alerts.

Comment (dhei) [reply appcenter-link-scripts/package.json]: I checked TestApp's package-lock.json and didn't find any reference to the insecure version of the package.

Comment (guperrot) [reply appcenter-link-scripts/package.json]: So, did someone test react native link on a blank app RN 0.59?

Comment (dhei) [reply appcenter-link-scripts/package.json]: Tested with RN 0.59.9 blank app.

**Comment by dhei** [comment]
Addressing AB#67835.

Tested with TestApp RN 0.60.x and blank app on RN 0.59.x.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,10 @@
 
 ## Version 2.5.0 (Under development)
 
+### App Center
+
+* **[Fix]** Fix dependency vulnerabilities in appcenter packages.
```
Comment (MatkovIvan) [inline CHANGELOG.md]:
```suggestion
* **[Fix]** Fix dependency vulnerabilities in App Center dependencies.
```

**Comment by michaelsmitha** [comment]
Is this linked to the correct work item? Wouldn't want to close the wrong one on accident. @russelarms @dhei

File: CHANGELOG.md (hunk 2 above)
Comment (dhei) [reply CHANGELOG.md]: @MatkovIvan By "appcenter packages" I mean all 6 AC RN SDK packages.
```suggestion
* **[Fix]** Fix dependency vulnerabilities in all App Center packages.
```

**Comment by dhei** [comment]
Thanks @michaelsmitha, fixed the link.

**Pull request merged**